numba/tests/test_np_functions.py

```diff
@@ -2926,6 +2930,96 @@ def check_window(func):
                 np_nbfunc(5, beta)
             self.assertIn("beta must be an integer or float", str(raises.exception))
 
+    def test_cross(self):
+        pyfunc = np_cross
+        cfunc = jit(nopython=True)(pyfunc)
+        pairs = [
+            # 3x3 (n-dims)
+            (
+                np.array([[1, 2, 3], [4, 5, 6]]),
+                np.array([[4, 5, 6], [1, 2, 3]])
+            ),
+            # 2x3 (n-dims)
+            (
+                np.array([[1, 2, 3], [4, 5, 6]]),
+                np.array([[4, 5], [1, 2]])
+            ),
+            # 3x3 (1-dim)
+            (
+                np.array([1, 2, 3]),
+                np.array([4, 5, 6])
+            ),
+            # 2x3 (1-dim)
+            (
+                np.array([1, 2]),
+                np.array([4, 5, 6])
+            ),
+            # 3x3 (with broadcasting 1d x 2d)
+            (
+                np.array([1, 2, 3]),
+                np.array([[4, 5, 6], [1, 2, 3]])
+            ),
+            # 3x3 (with broadcasting 2d x 1d)
+            (
+                np.array([[1, 2, 3], [4, 5, 6]]),
+                np.array([1, 2, 3])
+            )
```

Comment:
I think this test input selection would benefit from the addition of:

* higher order arrays (to check broadcasting etc)
* different and varying dtypes (to check the promotion logic)
* array-like input

Reply:
Will do

Reply:
Thanks, is there an array-like input test case somewhere? i.e. tuples or list input.

Reply:
cancel that, just spotted it :)